src/libraries/Directory.Build.targets

```diff
@@ -127,6 +127,11 @@
     <PackageReference Include="Microsoft.DotNet.GenFacades" Version="$(MicrosoftDotNetGenFacadesVersion)" PrivateAssets="all" IsImplicitlyDefined="true" />
   </ItemGroup>
 
+  <!-- TODO: Remove when all required nuget pack features are part of the consumed SDK. -->
+  <ItemGroup Condition="'$(IsPackable)' == 'true' and '$(MSBuildProjectExtension)' != '.pkgproj'">
+    <PackageReference Include="NuGet.Build.Tasks.Pack" Version="$(NuGetBuildTasksPackVersion)" />
```

Comment:
Is Implicitly Defined and PrivateAssets=all missing

Reply:
@Anipik can you please fix this? Otherwise the version would be controllable per package (in the package explorer) and it would show up as a dependency when creating a package via the NuGetPack task.

Reply:
Also, this package is only necessary for source projects, right? I  would change the condition to `$(IsPackable)' == 'true'  and '$(IsSourceProject)' == 'true'".